stable/mlrun/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mlrun.apiName" . }}
```

Comment:
what's with the `api` thing? there aren't many subcomponent here api, other... everything here is the api...?
If some things are related to api, and some to other - > move resource to subfolders `api` `whatever` but it looks flat to me, one deployment, one service....in which case `api` is redundant

Reply:
Talked offline there's UI component also
about the templates folder structure I've looked on other official helm charts like prometheus (https://github.com/helm/charts/tree/master/stable/prometheus) for example looks like the pattern is to do `api-deployment.yaml` `api-service.yaml` etc... and not api folder

Reply:
yeah, that's not very consistent. I've seen both, and I've seen a mixture of both (in places like pipelines). I think the folder approach is more friendly